core/README.md

```diff
@@ -161,43 +158,69 @@ Introduce your idea on the [stellar-dev mailing list](https://groups.google.com/
 Draft a formal proposal using the [CAP Template](../cap-template.md), and submit a PR to this
 repository. You should make sure to adhere to the following:
 
-* Use the following format for the filename of your draft: `cap_{github_username}_{shortsha256sum}.md`
-  * `shortsha256sum` is defined as the first 8 characters of the SHA-256 checksum.
-  * For example, a CAP by Github user `cryptocarnage` with a SHA-256 checksum of `a200f73c`
-    would be titled `cap_cryptocarnage_a200f73c.md`.
 * Make sure to place the draft in the `core/` folder.
+* Your CAP should be named `cap-TBD.md`
+* If your CAP requires images or other supporting files, they should be included in a sub-directory
+  of the `contents` folder for that CAP, such as `contents/cap-TBD/`. Links
+  should be relative, for example a link to an image from your CAP would be
+  `../contents/cap-TBD/image.png`.
 
 Finally, submit a PR of your draft via your fork of this repository.
 
 #### Additional Tips
 * Use `TBD` for the protocol version. Don't assign a protocol version to the CAP — this will be
   established once the CAP has reached the state of *Final* and has been formally implemented.
-* If your CAP requires images or other supporting files, they should be included in a sub-directory
-  of the `contents` folder for that CAP, such as `contents/cap_cryptocarnage_a200f73/`. Links
-  should be relative, for example a link to an image from your CAP would be
-  `../contents/cap_cryptocarnage_a200f73/image.png`.
 
 ### Draft: Merging & Further Iteration
-From there, the following process will happen:
-* A CAP buddy is assigned and will merge your PR if you properly followed the steps above.
-  * They'll rename the above files to the latest CAP draft number before merging in the PR.
-  * They'll provide initial feedback, and help pull in any subject matter experts that will help in
-    pushing the CAP towards a final disposition.
-* You should continue the discussion of the draft CAP on the mailing list with an attempt at
-  reaching consensus. We welcome any additional PRs that iterate on the draft.
+From there, the following process will happen.
+
+#### CAP gets merged
+If you properly followed the steps above, your PR will get merged.
+
+The CAP and associated files will get renamed based on the latest
+CAP draft number before merging.
+
+#### Assembling a working group
+As your idea gets traction, you'll need to assemble a working group as
+to increase the chances of success that this CAP proceeds through the stages.
+
+The recommended structure of the working group is based on the [RACI](https://en.wikipedia.org/wiki/Responsibility_assignment_matrix#Role_distinction) model.
+
+The template contains the following roles:
+  * Recommender - group of people that author the CAP with the owner
+  * Owner - maps to "Accountable" in RACI, the person responsible for the CAP
```

Comment:
Our description here of owner where it says the person is responsible for the CAP is ambiguous to me because in RACI the person responsible for the CAP is the recommender. We should probably avoid the word responsible if we can.

Reply:
done